src/Network/Network/help/Set-AzExpressRoutePortIdentity.md

```diff
@@ -0,0 +1,131 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Network.dll-Help.xml
+Module Name: Az.Network
+online version: https://docs.microsoft.com/en-us/powershell/module/az.network/set-azexpressrouteportidentity
+schema: 2.0.0
+---
+
+# Set-AzExpressRoutePortIdentity
+
+## SYNOPSIS
+Updates a identity assigned to an ExpressRoutePort.
+
+## SYNTAX
+
+```
+Set-AzExpressRoutePortIdentity -ExpressRoutePort <PSExpressRoutePort> -UserAssignedIdentityId <String>
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
```

Comment:
Cmdlet doesn't support `ShouldProcess`, please remove `WhatIf` and `Confirm` from readme.

Reply:
Done